app/views/shared/_footer.html.erb

```diff
@@ -1,18 +1,31 @@
 <footer class="navbar navbar-inverse site-footer">
   <div class="container-fluid">
-    <div class="navbar-text text-left">
-      <p><%= t('hyrax.footer.service_html') %></p>
-      <% if can? :read, :admin_dashboard %>
-        <p><a href="https://oregondigital.github.io/oregon-digital-style-guide/">Style Guide</a></p>
-      <% end %>
+    <div class="row">
+      <div class="col-md-offset-4 col-md-6 col-sm-8 col-sm-offset-4 white-text">
+        <%= link_to "About Oregon Digital", Hyrax::Engine.routes.url_helpers.help_path, method: :get, class: "white-text right-border" %>
+        <%= link_to "Policies", Hyrax::Engine.routes.url_helpers.help_path, method: :get, class: "white-text right-border" %>
+        <%= link_to "Copyright", Hyrax::Engine.routes.url_helpers.help_path, method: :get, class: "white-text right-border" %>
+        <%= link_to "Contact Us", Hyrax::Engine.routes.url_helpers.help_path, method: :get, class: "white-text right-border" %>
+        <%= link_to "Donate", Hyrax::Engine.routes.url_helpers.help_path, method: :get, class: "white-text" %>
+      </div>
     </div>
-    <div class="navbar-right">
-      <div class="navbar-text text-right">
-        <p class='deployed-version'>
-          Oregon Digital <%= "#{OregonDigital::VERSION}-#{ENV.fetch('DEPLOYED_VERSION', 'development')}" %>
-        </p>
-        <p>Powered by Hyrax <%= Hyrax::VERSION %></p>
-        <p><%= t('hyrax.footer.copyright_html') %></p>
+    <div class="row footer-row">
+      <div class="navbar-text text-left col-sm-4 col-xs-6">
+        <img class='osu-logo' src='<%= image_url('osu-logo.png')%>' alt="">
```

Comment:
`alt="Oregon State University"`

Reply:
These will eventually be links I assume, and will need alt-text since they won't include text in the link.
Regardless, these images contain meaningful text that needs to be alted.